src/astro/utils/load_dataframe.py

```diff
@@ -44,6 +49,12 @@ def move_dataframe_to_sql(
             schema=schema,
             warehouse=warehouse,
         ),
+        "google_cloud_platform": BigQueryHook(
+            gcp_conn_id=conn_id, use_legacy_sql=False
+        ),
+        "bigquery": TempBigQueryHook(
+            bigquery_conn_id=conn_id, use_legacy_sql=False, gcp_conn_id=conn_id
+        ),
```

Comment:
What is the difference here?

Reply:
Ohh we don't need 'google_cloud_platform' removing it.